elasticsearch/index_templates/common.settings.operations.orphaned.json

```diff
@@ -0,0 +1,11 @@
+{
+  "order": 5,
+  "settings": {
+    "index.refresh_interval": "5s",
+    "index.number_of_replicas": $REPLICA_SHARDS,
```

Comment:
I think you should use something else as a delimiter here for the token - `$REPLICA_SHARDS` makes it look too much like a shell variable - suggest using `%REPLICA_SHARDS%` with the token delimiter at the front and back.